clang/test/CodeGenSYCL/mark-kernel-name.cpp

```diff
@@ -1,14 +1,19 @@
-// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-host -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
-// RUN: %clang_cc1 -triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -triple x86_64-linux-pc -fsycl-is-host -disable-llvm-passes -fdeclare-spirv-builtins -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang_cc1 -triple spir64-sycldevice -aux-triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
```

Comment:
```suggestion
// RUN: %clang_cc1 -triple spir64 -aux-triple x86_64-linux-pc  -fsycl-is-device -disable-llvm-passes -emit-llvm %s -o - | FileCheck %s
```
nit: `sycldevice` should be "environment" component of the triple, so it seems to be unnecessary here.
Moreover we are planning to remove it soon, so it would be good if we can avoid adding it.